css-two-column-layout/styles.css

```diff
@@ -0,0 +1,81 @@
+h1,
+h3,
+button {
+  font-family: 'Karla', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+}
+
+h1 {
+  text-align: center;
+  letter-spacing: 3px;
+  margin-top: 55px;
+  margin-bottom: 15px;
+  font-size: 40px;
+}
+
+.head {
+  text-align: center;
+  padding-left: 225px;
+  padding-right: 225px;
+  padding-bottom: 45px;
+}
+
+p {
+  font-family: 'Karla', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: rgb(100, 99, 99);
+  font-size: 12px;
+  line-height: 20px;
+}
+
+h3 ~ p {
+  padding-right: 95px;
+  padding-left: 50px;
+}
+
+h3 {
+  padding-left: 50px;
+  padding-right: 140px;
+  font-size: 13px;
+  letter-spacing: 1px;
+}
+
+.row {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  padding-bottom: 25px;
+}
+
+.column-half {
+  flex-basis: 50%;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+button {
+  background-color: white;
+  color: rgb(218, 104, 62);
+  box-sizing: border-box;
+  padding: 7px 35px;
+  border: 2px solid rgb(218, 104, 62);
+  position: relative;
+  left: 400px;
```

Comment:
Instead of doing this large movement, you should set `text-align: center` on the parent element for this button.